Source/BoundaryConditions/MOSTStress.H

```diff
@@ -957,7 +959,7 @@ struct moeng_flux
             // Note: Kh = eta/rho
             //      hfx = -Kh dT/dz  ==> +ve hfx corresponds to heating from the surface
             // Extrapolate from klo to ghost cell a distance of -deltaz; negative signs cancel
-            dest_arr(i,j,k,icomp+n) = rho*(theta + moflux*rho/eta*deltaz);
+            dest_arr(i,j,k,icomp+n) = dest_arr(i,j,zlo,icomp+n) + moflux*rho/eta*deltaz;
```

Comment:
I like that this gets rid of the multiple factors of `rho`, which almost makes me do a double take.